src/dlgAliasMainArea.h

```diff
@@ -35,6 +35,9 @@ class dlgAliasMainArea : public QWidget, public Ui::aliases_main_area
 
 public:
     dlgAliasMainArea(QWidget*);
+    void trimName();
```

Comment:
Missed this the first time around - there's no need for anyone else but the class itself to call this function, so move it over to a `private:` section.

Reply:
that's where I started. but if you go and click on the toolbar buttons the finishEditing event doesn't fire. that's why I'm calling the public function in `saveScript()` etc.

Reply:
Makes sense - let's add that in as a comment so we remember the "why"